config.py

```diff
@@ -33,8 +33,8 @@
 
 # Downloading settings.
 area = "65/-20/30/20"
-upper_level = 122
+upper_level = 112
 
 # Processing settings.
-output_file_name = "results/processed_data_{:d}_{:d}.nc".format(start_year, final_year)
-read_n_lats_at_once = 1
\ No newline at end of file
+output_file_name = "/cephfs/user/s6lathim/ERA5Data-112/results/processed_data_europe_{start_year:d}_{final_year:d}_subset_{lat_subset_id:04d}_of_{max_lat_subset_id:04d}.nc"
```

Comment:
ah sorry, seems I missed that one. 

That would also be possible (if you remember, that's what I did before) - maybe lets see if we want that or not when talking in person, as it would probably make the discussion much faster.

Reply:
what do you mean with encoded? putting the region in or not is up to the user.